App/Sagas/NodeLifecycle.ts

```diff
@@ -4,25 +4,35 @@ import { ActionType, getType } from 'typesafe-actions'
 import RNFS from 'react-native-fs'
 import Config from 'react-native-config'
 import BackgroundTimer from 'react-native-background-timer'
+import PushNotification from 'react-native-push-notification'
 
-import TextileNodeActions, { TextileNodeSelectors, NodeState } from '../Redux/TextileNodeRedux'
+import TextileNodeActions, { NodeState, TextileNodeSelectors } from '../Redux/TextileNodeRedux'
+import { PreferencesSelectors } from '../Redux/PreferencesRedux'
 import TextileNode from '../../TextileNode'
 import { RootAction } from '../Redux/Types'
 import { Threads, ThreadName } from '../Models/TextileTypes'
 
 export function * manageNode () {
   while (true) {
+    const verboseUi: boolean = yield select(PreferencesSelectors.verboseUi)
     try {
       // Block until we get an active or background app state
       const action: ActionType<typeof TextileNodeActions.appStateChange> =
         yield take((action: RootAction) =>
           action.type === getType(TextileNodeActions.appStateChange) && (action.payload.newState === 'active' || action.payload.newState === 'background')
         )
 
-      // Get our current node state and create/start the node if it doesn't exist or is stopped
+      if (verboseUi) {
+        yield call(displayNotification, 'App State Change: ' + action.payload.newState)
+      }
+
+      // Get our current node state and create/start the node if it isn't started
       // Use fork so we don't block listening for the next app state change while the node is created and started
       const nodeState: NodeState = yield select(TextileNodeSelectors.nodeState)
-      if (nodeState === NodeState.nonexistent || nodeState === NodeState.stopped) {
+      if (nodeState !== NodeState.started) {
```

Comment:
This is the possible bug. Node wouldn't be started if it were in some transitioning state like `stopping`.